.zenodo.json

```
@@ -43,6 +53,10 @@
         "name": "Clark, Stephen Eric",
         "orcid": "0000-0002-0117-776X"
       },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory, now DESY",
+        "name": "Dammak, Eya"
```

```suggestion
        "name": "Dammak, Eya",
        "orcid": "0009-0003-7821-3508"
```
@EyaDammak orcid: https://orcid.org/0009-0003-7821-3508

---

Thank you! Added